packages/editor-ui/src/constants.ts

```diff
@@ -42,6 +42,7 @@ export const WORKFLOW_ACTIVE_MODAL_KEY = 'activation';
 export const ONBOARDING_CALL_SIGNUP_MODAL_KEY = 'onboardingCallSignup';
 export const COMMUNITY_PACKAGE_INSTALL_MODAL_KEY = 'communityPackageInstall';
 export const COMMUNITY_PACKAGE_CONFIRM_MODAL_KEY = 'communityPackageManageConfirm';
+export const IMPORT_CURL = 'importCurl';
```

Comment:
Nitpick: All modal id constants should end with `_MODAL_KEY` so they are easily recognizable.